src/onegov/org/models/political_business.py

```diff
@@ -189,6 +209,15 @@ def fts_suggestion(self) -> list[str]:
         'RISParliamentaryGroup',
         back_populates='political_businesses'
     )
+    # ---- no longer needed after ogc-2816 is in production
+
+    #: parliamentary groups (Fraktionen)
+    parliamentary_groups: relationship[list[RISParliamentaryGroup]]
+    parliamentary_groups = relationship(
+        'RISParliamentaryGroup',
+        secondary=par_political_business_parliamentary_groups,
+        back_populates='political_businesses',
```

Comment:
```suggestion
        back_populates='political_businesses',
        passive_deletes=True
```
If we have `ON DELETE CASCADE` on the relationship, we can make the deletes passive, rather than SQLAlchemy having to emit a manual delete statement.